### DISCUSSION TITLE
Syncing Shared Folders From Other User Accounts in The Same Organization

### DISCUSSION THREAD

**Discussion by adam2392** [abraunegg/onedrive#1444] [Q&A] [answered]
Hi 

Sorry for posting in a closed issue. Just reposting here based on https://github.com/abraunegg/onedrive/issues/459#issuecomment-833822849

I ran the command `onedrive --list-shared-folders`, and then I get;

```
Error: Unsupported account type for listing OneDrive Business Shared Folders
```

Do you happen to know if that's a result of my institutions OneDrive business account, or something config?

**Comment by abraunegg** [comment, marked as answer]
@adam2392 
This response indicates you are using a 'personal' account, not your corporate account / OneDrive Business Account to access the OneDrive Business Shared Folders.

You cannot do this when using a 'personal' account type.
